index.html

```diff
@@ -14,252 +14,400 @@
-        >
+        <ul class="images">
+          <li>
+            <img
+              src="img/Snapshoot Portfolio (1).png"
+              alt="card"
+              class="desktop-images"
+            />
+          </li>
+          <li>
+            <img
+              src="img/Snapshoot Portfolio (3).png"
+              alt="card"
+              class="mobile-images"
+            />
+          </li>
+        </ul>
       </section>
-
+    </main>
+    <section class="container">
       <section class="about-section">
-        <h2 class="myself">
-          About<br />
-          Myself
-        </h2>
-        <p class="career">
-          Hello I’m a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don’t hestiate to
-          contact me.
-        </p>
-        <h4 class="connect">LET’S CONNECT</h4>
-
-        <ul class="social-icons">
-          <li><span class="iconify" data-icon="iconoir:github"></span></li>
-          <li><span class="iconify" data-icon="eva:linkedin-fill"></span></li>
-          <li><span class="iconify" data-icon="fontisto:angelist"></span></li>
-          <li><span class="iconify" data-icon="ei:sc-twitter"></span></li>
-          <li><span class="iconify" data-icon="bx:bxl-medium"></span></li>
-        </ul>
-
-        <button class="resume">Get my resume</button>
+        <div class="career-container">
+          <h2 class="myself">
+            About<br />
+            Myself
+          </h2>
+          <p class="career">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <h4 class="connect">LET’S CONNECT</h4>
+          <div class="social-career">
+            <ul>
+              <li>
+                <a href=""
+                  ><span class="iconify" data-icon="iconoir:github"></span>
+                </a>
+              </li>
+              <li>
+                <a href=""
+                  ><span class="iconify" data-icon="eva:linkedin-fill"></span>
+                </a>
+              </li>
+              <li>
+                <a href=""
+                  ><span class="iconify" data-icon="fontisto:angelist"></span>
+                </a>
+              </li>
+              <li>
+                <a href=""
+                  ><span class="iconify" data-icon="ei:sc-twitter"></span>
+                </a>
+              </li>
+              <li>
+                <a href=""
+                  ><span class="iconify" data-icon="bx:bxl-medium"></span>
+                </a>
+              </li>
+            </ul>
+          </div>
+          <button class="resume">Get my resume</button>
```

Comment:
- [ ] [OPTIONAL]: I strongly recommend you to give more space between small sections:

Yours | Figma
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/144576349-9e7be46d-2bd3-4734-96da-ced50cf99a5b.png)  | ![image](https://user-images.githubusercontent.com/77038610/144576119-9dc9c953-7a00-483b-83ab-a88da8c728b7.png)

Right now the footer part of this section looks a bit congested 😢 

- [ ] [OPTIONAL] And please pay attention to difference in button style 😉 
- [ ] [OPTIONAL]: I can see that you updated color of social links on header, however it looks like you forgot to do it for footer 😁 . Please consider setting the color of these links to `#505F79`